public/app/features/browse-dashboards/state/hooks.ts

```diff
@@ -97,7 +104,7 @@ export function useActionSelectionState() {
   return useSelector((state) => selectedItemsForActionsSelector(state));
 }
 
-export function useLoadNextChildrenPage() {
+export function useLoadNextChildrenPage(loadDashboards = true) {
```

Comment:
dya think we could standardise on `excludedKinds` here? feels weird to see `loadDashboards` when you've been so careful to be generic with `kind` elsewhere 🤔

Reply:
Good shout - I missed this when I did the `excludedKinds`. I'll check how it feels.

Reply:
Check this out @ashharrison90 and lemme know what you think https://github.com/grafana/grafana/pull/71489/commits/4e65aef5831fa76c2dbf8ac8b94d52da241a9df3